ur_description/urdf/ur3_robot.urdf.xacro

```diff
@@ -2,25 +2,18 @@
 <robot xmlns:xacro="http://wiki.ros.org/xacro"
        name="ur3" >
 
-  <xacro:arg name="transmission_hw_interface" default="hardware_interface/PositionJointInterface"/>
-
-  <!-- common stuff -->
-  <xacro:include filename="$(find ur_description)/urdf/common.gazebo.xacro" />
-
-  <!-- ur3 -->
-  <xacro:include filename="$(find ur_description)/urdf/ur3.urdf.xacro" />
-
-  <!-- arm -->
-  <xacro:ur3_robot prefix="" joint_limited="false"
-    transmission_hw_interface="$(arg transmission_hw_interface)"
-  />
-
-  <link name="world" />
```

Comment:
I think, we shouldn't remove this in the standalone file. Isn't it quite common practice to have a x_robot.urdf.xacro with a world link in there?

At least for gazebo this seems to make a huge difference, as only then the parameter to lift the robot into the air seems to be evaluated. With the current implementation the gazebo models start shaking immediately.

Reply:
the issue here is that the file used to mix "being" a regular top-level xacro file with adding Gazebo compatibility.

As discussed this afternoon, it'd be best if the Gazebo-specific parts here are all moved to a separate set of files, and then using `gazebo` link references to add additional properties where needed.

Having a `world` link in this file can actually be quite problematic. See also #284.

Reply:
thanks for pointing that out :)

Reply:
I agree with @gavanderhoorn but the PR starts to be a bit sized, should we move to a dedicated  branch "reshape urdf" to handle everything on the reshape of the urdf ?

Reply:
Then we should update the gazebo launch files accordingly. With the current situation the robot is standing on the ground which will cause it to have collisions.

In the launchfile for ur_gazebo we add a parameter `-z 0.1` to lift it from the ground, but that only seems to work if a world link is present (correct me, if I'm wrong here).

So we could introduce a file `ur3_robot_gazebo.urfd.xacro` with the world link present?